tests/unit/commands/local/start_api/test_cli.py

```diff
@@ -96,6 +97,17 @@ def test_must_raise_user_exception_on_invalid_sam_template(self, invoke_context_
         expected = "bad template"
         self.assertEquals(msg, expected)
 
+    @patch("samcli.commands.local.start_api.cli.InvokeContext")
+    def test_must_raise_user_exception_on_invalid_env_vars(self, invoke_context_mock):
```

Comment:
It's almost like we should have a Test Suite for these kinds of common errors across similar commands. Will make these kinds of tests parametrizable instead of copying across tests. This is out-of-scope for this PR but something I noticed so capturing here.